main.go

```diff
@@ -206,6 +222,10 @@ func (configs ConfigsModel) validate() error {
 		}
 	}
 
+	if err := validateTimeout(configs.TestTimeout); err != nil {
```

Comment:
maybe it would make sense to remove the trailing `s`, may the user inputs `5s`, synchronize it with the step input description and add test for it.